cloud/lib/resourceNamingUtils.ts

```diff
@@ -0,0 +1,15 @@
+import { Construct } from 'constructs';
+
+export const appName = 'SpyLogic';
+
+export const stageName = (construct: Construct) =>
+	(construct.node.tryGetContext('STAGE') as string) || 'dev';
+
+export const resourceName = (construct: Construct) => (suffix: string) =>
+	`${stageName(construct)}-${appName}-${suffix}`.toLowerCase();
+
+export const resourceDescription = (construct: Construct) => (prefix: string) =>
+	`${prefix} for ${appName} (${stageName(construct)})`;
```

Comment:
Seems you were hungry for some Curry when you were writing this.

Any particular reason for things to be defined this way? Convention? Speed?